pokemongo_bot/cell_workers/follow_cluster.py

```diff
@@ -42,7 +42,7 @@ def work(self):
 
                 log_str = log_lure_avail_str + 'Move to destiny. ' + str(cnt) + ' ' + log_lured_str + \
                           'pokestops will be in range of ' + str(self.radius) + 'm. Arrive in ' \
-                          + str(distance(self.bot.position[0], self.bot.position[1], lat, lng)) + 'm.'
+                          + str(round(distance(self.bot.position[0], self.bot.position[1], lat, lng))) + 'm.'
```

Comment:
The commits would need to be on different branches. It is a lot of work for us to split up your PR. Since this is such a small change it would be easiest for you to just add a new commit undoing this change, and then on a new branch adding it again.